src/components/AnalysisPage/GeoAnalysisBox/StartButton/pollForResults.js

```diff
@@ -0,0 +1,17 @@
+export const pollForResults = async (jobId, setIsLoading, setGeoData) => {
+  console.log("polling...");
+  const response = await fetch(`/api/v0/geometry/${jobId}`);
+  if (!response.ok) {
+    throw new Error("Failed to fetch analysis results");
+  }
+
+  const results = await response.json();
+
+  if (results.status === "complete") {
+    setIsLoading(false);
+    setGeoData(results);
```

Comment:
Gotchya! 

I thought we were only making sure to keep the 3d model viewer react-less. 

But all http request code should stay away from react as well, understood

Reply:
Yea its not a huge deal but I think its just generally a good practice to keep like 'pure' javascript stuff separate from React javascript stuff. It can just make it easier if you wanted to just test out the http request code by itself without needing to have React stuff passed in